src/content/docs/extensions/json.mdx

```diff
@@ -515,4 +515,315 @@ and `"to"` in the file, whose values point to the primary key values of the unde
 JSON data being copied, as well as dynamically transform your data via `UNWIND` or `DISTINCT` clauses,
-so it's not necessary to write your relationships to an intermediate file prior to using `COPY`.
\ No newline at end of file
+so it's not necessary to write your relationships to an intermediate file prior to using `COPY`.
+
+## JSON Type
+KuzuDB supports `json` via the `JSON` logical type. The `JSON` logical type is interpreted as parsed JSON, 
+rather than interpreted as `string`.
+
+The following example creates a node table `person` with a `JSON` column `description`,
+it then creates two `json` objects in this column using `to_json` function, and outputs them.
+
+Example:
+```cypher
+CREATE NODE TABLE person (id INT64, description json, primary key(id));
+CREATE (p:person {id: 20, description: to_json({height: 52, age: 32, scores: [1,2,5]})});
+CREATE (p:person {id: 40, description: to_json({age: 55, scores: [1,32,5,null], name: 'dan'})});
+MATCH (p:person) RETURN p.*;
+```
+
+```json
+20|{"height":52,"age":32,"scores":[1,2,5]}
+40|{"age":55,"scores":[1,32,5,null],"name":"dan"}
+```
+
+
+## JSON functions
+
+This section lists the built-in functions that operate on the JSON data type within Kùzu.
+
+### `to_json`
+
+Signature: `ANY -> JSON`
+
+`to_json(any)`
+
+Converts `any` Kùzu value to `JSON`.
+
+Examples: 
+
+```cypher
+RETURN to_json('kuzu');
+```
+Output:
+```json
+"kuzu"
+```
+
+```cypher
+RETURN to_json([1,2,3]);
+```
+Output:
+```json
+[1,2,3]
+```
+
+```cypher
+RETURN to_json('{"name": "Alicia", "age": 28}');
+```
+Output:
+```json
+{"name":"Alicia","age":28}
+```
+
+### `array_to_json`
+
+Signature: `ARRAY -> JSON`
+
+Alias for `to_json` that only accepts `ARRAY`.
+
+### `row_to_json`
+
+Signature: `LIST -> JSON`
+
+Alias for `to_json` that only accepts `LIST`.
+
+### `json_object`
+
+Signature: `STRING, ANY -> JSON object`
+
+`json_object([key, value, ...])`
+
+Create a JSON object from any number of `key`, `value` pairs.
+
+Examples:
+
+```cypher
+RETURN json_object("name","Alicia");
+```
+```json
+{"name":"Alicia"}
+```
+
+```cypher
+RETURN json_object("name","Alicia","age",28);
+```
+```json
+{"name":"Alicia","age":28}
+```
+
+### `json_array`
+
+Signature: `ARRAY -> JSON ARRAY`
+
+`json_array([any, ...])`
+
+Create a JSON array from `any` number of values.
+
+Examples:
+
+```cypher
+RETURN json_array(42, 'kuzu', NULL);
+```
+```json
+[42,"kuzu",null]
+```
+
+
+### `json_merge_patch`
+
+Signature: `JSON, JSON -> JSON`
+
+`json_merge_patch(json, json)`
+
+Merges TWO `JSON` documents. Applies [RFC 7386](https://datatracker.ietf.org/doc/html/rfc7386)
+
+Examples:
+
+```cypher
+RETURN json_merge_patch('{"name": "Alicia"}', '{"age": 28}');
+```
+Output:
+```json
+{"name":"Alicia","age":28}
+```
+```cypher
+RETURN json_merge_patch('3', NULL);
+```
+Output:
+```json
+```
+(merge with a NULL path would result in NULL path)
+
+### `json_extract`
+
+Signatures: `JSON, STRING -> JSON`, `JSON, INTEGER -> JSON`
```

Comment:
I think we also support multipath json extract like:
```
json_extract(j, ['family', 'species']) 
```

Reply:
added the example query for multipath